src/Sylius/Bundle/ApiBundle/Resources/config/app/config.yaml

```diff
@@ -10,22 +10,33 @@ parameters:
-        - "%sylius.security.new_api_route%/admin/catalog-promotion-scopes/{id}"
-        - "%sylius.security.new_api_route%/admin/channel-pricings/{id}"
-        - "%sylius.security.new_api_route%/admin/promotion-actions/{id}"
-        - "%sylius.security.new_api_route%/admin/promotion-rules/{id}"
-        - "%sylius.security.new_api_route%/admin/shipping-method-rules/{id}"
-        - "%sylius.security.new_api_route%/admin/shop-users/{id}"
-        - "%sylius.security.new_api_route%/shop/adjustments/{id}"
-    sylius.security.new_api_route: "/api/v2"
-    sylius.security.new_api_regex: "^%sylius.security.new_api_route%"
-    sylius.security.new_api_admin_route: "%sylius.security.new_api_route%/admin"
-    sylius.security.new_api_admin_regex: "^%sylius.security.new_api_admin_route%"
-    sylius.security.new_api_shop_route: "%sylius.security.new_api_route%/shop"
-    sylius.security.new_api_shop_regex: "^%sylius.security.new_api_shop_route%"
-    sylius.security.new_api_user_account_route: "%sylius.security.new_api_shop_route%/account"
-    sylius.security.new_api_user_account_regex: "^%sylius.security.new_api_user_account_route%"
+        - "%sylius.security.api_route%/admin/catalog-promotion-actions/{id}"
+        - "%sylius.security.api_route%/admin/catalog-promotion-scopes/{id}"
+        - "%sylius.security.api_route%/admin/channel-pricings/{id}"
+        - "%sylius.security.api_route%/admin/promotion-actions/{id}"
+        - "%sylius.security.api_route%/admin/promotion-rules/{id}"
+        - "%sylius.security.api_route%/admin/shipping-method-rules/{id}"
+        - "%sylius.security.api_route%/admin/shop-users/{id}"
+        - "%sylius.security.api_route%/shop/adjustments/{id}"
+
+    sylius.security.api_route: "/api/v2"
+    sylius.security.api_regex: "^%sylius.security.api_route%"
+    sylius.security.api_admin_route: "%sylius.security.api_route%/admin"
+    sylius.security.api_admin_regex: "^%sylius.security.api_admin_route%"
+    sylius.security.api_shop_route: "%sylius.security.api_route%/shop"
+    sylius.security.api_shop_regex: "^%sylius.security.api_shop_route%"
+    sylius.security.api_shop_account_route: "%sylius.security.api_shop_route%/account"
+    sylius.security.api_shop_account_regex: "^%sylius.security.api_shop_account_route%"
+
+    # Below parameters are deprecated since Sylius 1.14 and will be removed in 2.0.
+    sylius.security.new_api_route: "%sylius.security.api_route%"
+    sylius.security.new_api_regex: "%sylius.security.api_regex%"
+    sylius.security.new_api_admin_route: "%sylius.security.api_admin_route%"
+    sylius.security.new_api_admin_regex: "%sylius.security.api_admin_regex%"
+    sylius.security.new_api_shop_route: "%sylius.security.api_shop_route%"
+    sylius.security.new_api_shop_regex: "%sylius.security.api_shop_regex%"
+    sylius.security.new_api_user_account_route: "%sylius.security.api_shop_account_route%"
+    sylius.security.new_api_user_account_regex: "%sylius.security.api_shop_account_regex%"
```

Comment:
These should be reversed since the usage was swapped to the new keys, if I've overwritten etc. `sylius.security.new_api_admin_regex` in my app then this config will break my routing since the one actually in use is `sylius.security.api_admin_regex`